python/lsst/ip/isr/isrTask.py

```diff
@@ -366,6 +366,15 @@ class IsrTaskConfig(pipeBase.PipelineTaskConfig,
         doc="Number of edge pixels to be flagged as untrustworthy.",
         default=0,
     )
+    edgeMaskLevel = pexConfig.ChoiceField(
+        dtype=str,
+        doc="Which level to mask edge values at?",
```

Comment:
I feel like this could be a little more descriptive. Maybe something like "Mask edge pixels around the perimeter of the detector, or around each amplifier?" or something like that, otherwise it's a little tautological.